src/bidiMapper/domains/context/BrowsingContextImpl.ts

```diff
@@ -930,6 +932,31 @@ export class BrowsingContextImpl {
   }
 }
 
+function getImageFormatParameters(
+  params: Readonly<BrowsingContext.CaptureScreenshotParameters>
```

Comment:
Because this includes `context` which is not relevant to this function, I would make the type narrower: just pass two arguments:

`getImageFormatParameters(type: string, quality?: number)`

Reply:
This would require more checks on the parameters because type and quality are gated within the same object. It would just be more code.